alpakaConfig.cmake

```diff
@@ -512,18 +674,36 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
     MESSAGE(STATUS ALPAKA_ACC_GPU_CUDA_ENABLED)
 ENDIF()
 
+#------HIP enabled------
+IF(ALPAKA_ACC_GPU_HIP_ENABLE)
+    LIST(APPEND _ALPAKA_COMPILE_DEFINITIONS_PUBLIC "ALPAKA_ACC_GPU_HIP_ENABLED")
+    MESSAGE(STATUS ALPAKA_ACC_GPU_HIP_ENABLED)
+ENDIF()
+#-----------------------
 LIST(APPEND _ALPAKA_COMPILE_DEFINITIONS_PUBLIC "ALPAKA_DEBUG=${ALPAKA_DEBUG}")
 
 IF(ALPAKA_CI)
     LIST(APPEND _ALPAKA_COMPILE_DEFINITIONS_PUBLIC "ALPAKA_CI")
 ENDIF()
 
+#-----------------------
+# workaround to get the flags correctly to the hip compiler
+IF(ALPAKA_ACC_GPU_HIP_ENABLE)
+    SET(hip_defs ${_ALPAKA_COMPILE_DEFINITIONS_PUBLIC})
```

Comment:
Please write all variables upper case and prefixed by `ALPAKA` and additionally by `_` for internal ones to be consistent. (e.g. `_ALPAKA_COMPILE_DEFINITIONS_HIP`)